backend/services/requestService.ts

```diff
@@ -0,0 +1,20 @@
+import { CourtEmail } from 'whereaboutsApi'
+import type { WhereaboutsApi } from '../api'
+import { Context, RequestEmail } from './model'
+import type NotificationService from './notificationService'
+
+export = class RequestService {
```

Comment:
meh.. would generally just `export default`, the `=` is for when you need js interop